docs/config.json

```diff
@@ -567,17 +567,22 @@
           "slug": "/management/export-audit-events/",
           "entries": [
             {
-              "title": "Exporting Audit Events to Fluentd",
+              "title": "Forward Audit Events to Fluentd",
               "slug": "/management/export-audit-events/fluentd/",
               "forScopes": ["enterprise", "cloud"]
             },
             {
-              "title": "Monitoring Audit Events with the Elastic Stack",
+              "title": "Monitor Audit Events with Datadog",
```

Comment:
Is monitor a better verb than export here?

I ask because monitoring (especially in the context of our docs) is generally referring to Prometheus-style monitoring for performance metrics as opposed to audit events.

Reply:
I'm happy with either, I mostly wanted to removing the "ing" suffix. I made the change in words to match the other guides in this list.
Should they all be "Export" over "Monitor"?

Reply:
I prefer export if you're happy with either.

Reply:
Done.